the-devastating-chickens/BangazonSite
# Wlm select product category

# Description

Creating ProductsController with get and post methods to create a new product. Adding validation to necessary fields including productTypeId. Adding view in layout. 

[Fixes Issue #4](https://github.com/the-devastating-chickens/BangazonSite/issues/4)

## Type of change


# Testing Instructions

1. `git fetch --all`
1. `git checkout wlm-select-product-category`
1. Run Bangazon program
1. Click on *Sell a Product* link
1. Create a new product while verifying that all required fields must be entered.
1. Verify that the new product is displayed on the redirected product page.

# Checklist:
